### ISSUE TITLE
Baubles/Botania Crash

### ISSUE THREAD

**Issue by SolarTara** [Vazkii/Botania#2531]
Forge version: 1.12.2-14.23.1.2604-universal
Botania version: Botania-Mod-1.12.1 
Link to crash log: https://gist.github.com/SolarTara/383b58c30325b723923db101173f7a09

Steps to reproduce:
1. Equip a mana band in the babbles spot
2. Equip an invisibility cloak in the other babbles spot
3. Instant kick and crash

What I expected to happen:
  -Turn invisible and drain mana from the band

What happened instead:
   -Kicked from both a single player game and multiplayer game

**Comment by ScottNotFound** [comment]
I've found this happens with a lot more than just the invisibility cloak. The result is the same each time. The others I have found are the aura ring that generates mana, the soujourner's sash, and the ring of chordata (though this one only happens when the bubbles are gone and it activates the breathing ability). I also notice many errors in the logs relating to network packets and I am fairly sure they are strongly related.
